### PR TITLE
updates social login design doc

### PR DESCRIPTION
**Pull request by matthewcarleton**
## Description
This updates the social login page to show the brand guidelines for each logo used.

## Contribution Submission Checklist

- [ ] A GitHub issue was created and reviewed with the PatternFly community.
- [ ] The pattern design overview.md and design.md templates were used to provide all information, relevant to this design.
- [ ] Any new files have been added to the folder for that pattern.
- [ ] The folder, for brand new patterns, has been titled with the name of the pattern.
- [ ] All images are in an "img" folder and embedded images have the correct filepath.
- [ ] The overview.md, design.md and site.md files, for this design have been commited and pushed to the patternfly-design repository, for review.

### CODE REVIEW

**Comment by LHinson** [comment]
Very good. LGTM. 

@mcarrano: @matthewcarleton added this for legal purposes. If you approve, please merge!

**Comment by matthewcarleton** [comment]
ya good point @mcarrano, I'll fix the list too. Still new to MD editing :)

**Comment by matthewcarleton** [comment]
@mcarrano ok, updated. Please confirm and merge. Thanks!

**Comment by LHinson** [comment]
@matthewcarleton Want to clarify one thing... In order to create the example, we are including svgs for those specific logos in PatternFly, correct? 

If so, @mcarrano, this is why we are specifically calling out these logos because they are delivered with PF and want to cover our bases about usage since we are distributing them. Any other logos used in a product, separate from PF, is the product's responsibility.

File: pattern-library/application-framework/login-page/design/overview.md [hunk 1]
```diff
@@ -23,4 +23,15 @@ This is a general layout of a common login screen. A user can input the username
 ## Social Login
 Social login will allow a user to sign in with their social networks or IdPs accounts.
 
+
+Please refer to branding guidelines when using logos for social login page. The following are examples of some of the more frequently used but you may use others as your needs require.
+
+- [Facebook brand guidelines](https://en.facebookbrand.com/guidelines/brand)
+- [Twitter brand guidelines](https://about.twitter.com/en_us/company/brand-resources.html)
+- [Github brand guidelines](https://github.com/logos)
+- [stackexchange brand guidelines](https://stackexchange.com/legal/trademark-guidance)
```
Comment (mcarrano) [inline pattern-library/application-framework/login-page/design/overview.md]: Shouldn't this be "Stack Exchange..."?

**Comment by matthewcarleton** [comment]
thanks @mcarrano updated :)
@LHinson yup that is correct. We talked about also removing them and just adding generic icons, but I thought since there are the most common it would helpful to add them in.

**Comment by mcarrano** [comment]
Still seeing the work "stack" in lower case, i.e. "stack Exchange"

**Comment by matthewcarleton** [comment]
@mcarrano oh oops, fixed :)

**Comment by matthewcarleton** [comment]
@mcarrano ya, I saw that. Thanks

**Comment by mcarrano** [comment]
Looks like we will stick with the Logos and the text links, as included here. This probably also needs to be added to the SSO Login pattern since we also use the logos there, right? I also note that we show additional logo icons when the list is expanded on the design.md page. @LHinson Do we need to have a link for every icon we use or is it OK to just have a subset like this?

**Pull request merged**